<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingUI</class>
 <widget class="QWidget" name="SettingUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72, 72, 72);
color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane
{
	border-width:1px;border-color:rgb(48, 104, 151);
	border-style: outset;background-color: rgb(132, 171, 208);
	background: transparent;
} 
QTabWidget::tab-bar
{
	border-width:0px;
}
QTabBar::tab
{
	border-bottom-color:  rgb(72, 72, 72);
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
	max-width: 75px; min-width:75px; min-height:25px;
	padding: 0px;
}

QTabBar::tab:first:selected
{
	margin-top: -2px;
	color: white;
	border-width:1px;
	background-color: rgb(127, 127, 127);
}
QTabBar::tab:first:!selected 
{
	color: rgb(200, 200, 200);
	border-width:1px;
	background-color: rgb(100, 100, 100);
}
QTabBar::tab:first:hover:!selected 
{
	color: white;
	border-width:1px;
	background-color: rgb(200, 200, 200);
}
QTabBar::tab:middle:selected
{
	margin-top: -2px;
	color: white;
	border-width:1px;
	background-color: rgb(127, 127, 127);
}
QTabBar::tab:middle:!selected 
{
	color: rgb(200, 200, 200);
	border-width:1px;
	background-color: rgb(100, 100, 100);
}
QTabBar::tab:middle:hover:!selected 
{
	color: white;
	border-width:1px;
	background-color: rgb(200, 200, 200);
}
QTabBar::tab:last:selected
{
	margin-top: -2px;
	color: white;
	border-width:1px;
	background-color: rgb(127, 127, 127);
}
QTabBar::tab:last:!selected 
{
	color: rgb(200, 200, 200);
	border-width:1px;
	background-color: rgb(100, 100, 100);
}
QTabBar::tab:last:hover:!selected 
{
	color: white;
	border-width:1px;
	background-color: rgb(200, 200, 200);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>标签配置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pBtnAddColorLabel">
             <property name="text">
              <string>添加标签</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pBtnDelColorLabel">
             <property name="text">
              <string>删除标签</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>其他</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
